seq2seq/models/DecoderRNN.py

```diff
@@ -140,7 +165,8 @@ def forward_step(self, input_var, hidden, encoder_outputs, function):
         return predicted_softmax, hidden, attn
 
     def forward(self, inputs=None, encoder_hidden=None, encoder_outputs=None,
-                    function=F.log_softmax, teacher_forcing_ratio=0):
+                    function=F.log_softmax, teacher_forcing_ratio=0, **kwargs):
```

Comment:
Can you add here a description of the things that can potentially be in the keyword arguments? Is it just 'provided_guidance'?